run_dir/design/project_samples.html

```diff
@@ -18,8 +18,7 @@ <h1>
   <a href="#" class="btn btn-outline-dark btn-xs float-right" id="show_order_timeline" style="margin-top:20px;"><span id="show_orderdates_btn" style="display:none;">Show</span> <span id="hide_orderdates_btn">Hide</span> order dates on timeline</a>
```

Comment:
We also don't need the Show and hide buttons on this line.

Reply:
I noticed that too yesterday, already gone! (But not pushed yet) 👍